Task1:- Read and file and handel error
      Explanation:- This code is used for reading a file and also does exception handeling for the "FileNotFound" error
      this is done using the try, except block and using the finally keyword we print the final print statements.
      This is done to a file named file1.txt, which is present in the same location as the Task1.py is present (file1.py is also attached in the repo)
      <img width="777" height="93" alt="image" src="https://github.com/user-attachments/assets/f8803d0d-c048-441a-8fa4-b61d6c7b8180" />
      This is before the file was created.
      <img width="810" height="226" alt="image" src="https://github.com/user-attachments/assets/c1891c09-8bfe-4b26-a52c-5b1c5c83b0ba" />
      The text that is present in the file.
      <img width="784" height="113" alt="image" src="https://github.com/user-attachments/assets/1fa10e15-22f4-4ca8-baf9-524ef42bf9aa" />
      This is after the file is created.



